api/proto/bcconn/bcconn.go

```diff
@@ -19,6 +20,8 @@ type ResponseRandomNumber struct {
 // SerializeNodeInfo is for serializing nodeinfo
 func SerializeNodeInfo(nodeinfo *node.Info) []byte {
 	var result bytes.Buffer
+	fmt.Printf("SerializeNodeInfo: %s\n", *nodeinfo)
```

Comment:
```suggestion
	log.Debug("SerializeNodeInfo", "nodeinfo", *nodeinfo)
```

Reply:
The log.Debug won't print out the line I was expecting.  I think the log is not working properly here.